src/uct/tcp/tcp_md.c

```diff
@@ -82,7 +82,7 @@ uct_component_t uct_tcp_component = {
     .md_config          = UCT_MD_DEFAULT_CONFIG_INITIALIZER,
     .cm_config          = {
         .name           = "TCP-SOCKCM connection manager",
-        .prefix         = "TCP_",
+        .prefix         = "TCP_CM_",
```

Comment:
why needed?

Reply:
to distinguish between tcp tl params and tcp_sockcm params.

Reply:
let's remove, we do not need it on int3 branch as part of this PR

Reply:
updated.